src/styles/_catalog.scss

```
@@ -14,6 +14,15 @@
     border: 1px solid #ccc;
     border-radius: 5px;
     margin-bottom: 48px;
+
+    transition-duration: 300ms;
+    transition-property: transform, color;
```

It would be better to use short syntax in the future
```suggestion
    transition: transform 300ms, color 300ms;
```